bin/nodejs-wrapper.sh

```diff
@@ -51,7 +52,7 @@ setup_node_venv() {
     . "${python_venv_path}/bin/activate"
 
     echo "Installing node_env"
-    pip install nodeenv
+    python -m pip install nodeenv
```

Comment:
line 35 of same script uses `python3` vs naked `python` here.  Intentional?  (need comment as to why?)

Reply:
It's debian policy to keep `python` as `python2` ([and any attempt to change it will break the system](https://stackoverflow.com/questions/43062608/how-to-update-alternatives-to-python-3-without-breaking-apt/46095470#46095470)); this invocation (`python`) is from inside an activated virtual environment, which sets up `python` to point to `python3` as we would hope.

I'll change this line to `python3` to make it explicit. Hopefully debian will eventually switch :)